frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceModal.unit.spec.tsx

```diff
@@ -393,6 +393,25 @@ describe("ValuesSourceModal", () => {
 
       expect(screen.getByRole("textbox")).toHaveValue("Gadget\nWidget");
     });
+    it("should render a hint about using models when labels are used", async () => {
```

Comment:
nit: the newlines :pray: 

```suggestion

    it("should render a hint about using models when labels are used", async () => {
```